pkg/u-boot/rpi/config.txt

```diff
@@ -27,6 +27,9 @@ avoid_warnings=1
 # upstream provides
 dtoverlay=raspberrypi-rpi
 
+# TPM Module
+#dtoverlay=tpm-slb9670
```

Comment:
Presumably this should not be commented out.

Can it be there even on devices without a TPM without causing issues?

Reply:
I think so, but I need to check.
I don't have a TPM module - another person is testing. After he confirms that the TPM module is working, it will be clear the number of changes and whether they will affect devices without a TPM module.